### PR TITLE
Standalone God mode

### PR DESCRIPTION
**Pull request by chetanmeh**
Standalone OpenWhisk now supports quite a few optional features which can be enabled by  various options (`--kafka`, `--api-gw`, `--counchdb`, `--user-events`) etc. 

This PR introduces a `--all` mode which would enable all the above features via single argument and launch all optional services (Kafka, Prometheus, Redis etc)

## Description

Run all supported OpenWhisk Standalone features

```
java -jar openwhisk-standalone.jar --all
```

Currently this mode would launching following services (in 7 docker containers)

```
===============================================================
Launched service details

[ 3233  ] http://localhost:3233 (Controller)
[ 6379  ] http://localhost:6379 (whisk-redis)
[ 9000  ] http://localhost:9000 (whisk-apigw, Api Gateway - Api Service )
[ 3234  ] http://localhost:3234 (whisk-apigw, Api Gateway - Management Service)
[ 5984  ] http://localhost:5984 (whisk-couch)
[ 9092  ] localhost:9092 (kafka)
[ 9091  ] 192.168.65.2:9091 (kafka-docker)
[ 2181  ] Zookeeper (zookeeper)
[ 64231 ] http://localhost:64231 (whisk-kafka-drop-ui)
[ 3235  ] http://localhost:3235 (whisk-user-events)
[ 9090  ] http://localhost:9090 (whisk-prometheus)
[ 3000  ] http://localhost:3000 (whisk-grafana)
```

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [ ] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [ ] I reviewed the [style guides](https://github.com/apache/openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

File: core/standalone/src/main/scala/org/apache/openwhisk/standalone/StandaloneOpenWhisk.scala [hunk 1]
```diff
@@ -96,13 +95,37 @@ class Conf(arguments: Seq[String]) extends ScallopConf(arguments) {
 
   val userEvents = opt[Boolean](descr = "Enable User Events along with Prometheus and Grafana", noshort = true)
 
+  val all = opt[Boolean](
+    descr = "Enables all the optional services supported by Standalone OpenWhisk like CouchDB, Kafka etc",
```
Comment (selfxp) [inline core/standalone/src/main/scala/org/apache/openwhisk/standalone/StandaloneOpenWhisk.scala]: This is great, should we update `README` too? 
```
      --kafka-ui                   Enable Kafka UI

      --all                        Enables all the optional services supported
                                   by Standalone OpenWhisk like CouchDB, Kafka
                                   etc
```

Comment (chetanmeh) [reply core/standalone/src/main/scala/org/apache/openwhisk/standalone/StandaloneOpenWhisk.scala]: Updated the help docs

File: core/standalone/README.md [hunk 2]
```diff
@@ -75,29 +75,33 @@ $ java -jar openwhisk-standalone.jar -h
...
       --couchdb                    Enable CouchDB support
+      --user-events                Enable User Events along with Prometheus and
+                                   Grafana
+      --kafka                      Enable embedded Kafka support
+      --kafka-ui                   Enable Kafka UI
+
+      --all                        Enables all the optional services supported
+                                   by Standalone OpenWhisk like CouchDB, Kafka
+                                   etc
+      --api-gw-port  <arg>         Api Gateway Port
```
Comment (rabbah) [inline core/standalone/README.md]:
```suggestion
      --api-gw-port  <arg>         API Gateway Port
```

File: core/standalone/README.md [hunk 3]
```diff
@@ -75,29 +75,33 @@ $ java -jar openwhisk-standalone.jar -h
  \   \  /  \/    \___/| .__/ \___|_| |_|__/\__|_| |_|_|___/_|\_\
   \___\/ tm           |_|
 
-      --api-gw                     Enable API Gateway support
-      --api-gw-port  <arg>         Api Gateway Port
-      --clean                      Clean any existing state like database
+  -m, --manifest  <arg>            Manifest json defining the supported runtimes
```
Comment (rabbah) [inline core/standalone/README.md]:
```suggestion
  -m, --manifest  <arg>            Manifest JSON defining the supported runtimes
```

**Pull request merged**